makefile

```diff
@@ -32,11 +32,17 @@ endif
 
 CFLAGS := -fPIC
 
+# CUDA
 ifneq (,$(wildcard /usr/local/cuda/include/cuda.h))
 STACK_FLAGS = --flag dex:cuda
 CFLAGS := $(CFLAGS) -I/usr/local/cuda/include -DDEX_CUDA
 endif
 
+# libpng
+ifneq (,$(wildcard /usr/local/include/png.h))
+CFLAGS := $(CFLAGS) -I/usr/local/include
+endif
```

Comment:
Uh, this is not the way to go. If it's a homebrew-specific issue then you should use `brew --prefix`